app/src/main/java/com/algo/phantoms/algo_phantoms/fragments/CodeFragment.kt

```diff
@@ -23,9 +23,9 @@ class CodeFragment : Fragment() {
         val view = inflater.inflate(R.layout.fragment_code, container, false)
         binding = FragmentCodeBinding.bind(view)
         binding.editor.setSyntaxHighlightRules(
-                SyntaxHighlightRule("[0-9]*", "#6512c4"),
-                SyntaxHighlightRule("[^A-Za-z0-9]","#F30E0B"),
-                SyntaxHighlightRule("[A-Za-z]", "#242322")
+            SyntaxHighlightRule("[0-9]*", "#6512c4"),
+            SyntaxHighlightRule("[^A-Za-z0-9]","#F30E0B"),
+            SyntaxHighlightRule("[A-Za-z]",  "#242322")
```

Comment:
@priyanka0906 it always a good practise to user color from resources, instead of directly using them as color codes

Reply:
@carotkut94  I tried using colors from colors.xml but getColor() gives int value and the functions above expecting string value ...so it is showing error. I have tried taking string value from strings.xml its working fine ..can we do like that storing colors in strings.xml? would you suggest  any other way of doing it.